app/api/sessions.py

```diff
@@ -256,6 +256,13 @@ def before_update_object(self, session, data, view_kwargs):
             )
 
         new_state = data.get('state')
+
+        g.send_email = (
+            new_state is not None
+            and new_state != session.state
+            and (new_state == 'accepted' or new_state == 'rejected')
+        )
```

Comment:
This should be written inside the if condition below

Reply:
This also does not contain the data.get('send_email') logic

Reply:
So below I should write `(new_state == 'accepted' or new_state == 'rejected')` and `data.get('send_email', None)` apart from `g.send_email`?

Reply:
Yes